backend/tournesol/entities/base.py

```diff
@@ -186,6 +185,14 @@ def filter_date_lte(cls, qs, max_date):
     def filter_date_gte(cls, qs, min_date):
         return qs.filter(add_time__gte=min_date)
 
+    @classmethod
+    def get_allowed_meta_order_fields(cls) -> Optional[List[str]]:
+        """
+        Return a list of metadata fields that can be used to order an entity
+        list.
+        """
+        return None
```

Comment:
minor: would it make sense to use the same return type `List[str]` here and in child classes, and return `[]` by default?

Reply:
Sure, it will be more easy to read and understand.